Automattic/wp-calypso
# Stats: Create Placeholder Component

Currently the markup for rendering placeholder/skeleton loading indicators is scattered throughout all stats components.  This branch aims to centralize all of that markup/scss into one central location, and to fix some behaviors around it.

__To Test__
The best way to test the placeholders is to empty your localStorage on the Insights page, and a *year* view of a site stats page.  Refresh the pages on empty local storage and ensure everything still has pulsing loading indicators, and after 2s the spinners should appear.

__To Do__
- [x] Decrease time to show spinners fixes #2488 
- [x] Move logic of displaying placeholders out of css into component

/cc @alternatekev and @jancavan for thoughts
